installer/templates/assets/brunch/app.js

```diff
@@ -11,7 +11,7 @@
 //
 // If you no longer want to use a dependency, remember
 // to also remove its path from "config.paths.watched".
```

Comment:
Should i move the beginning of the `if` to this line, so it does not create 2 newlines?